src/main/java/com/artipie/api/ManageUsers.java

```diff
@@ -109,7 +119,40 @@ public void remove(final String uid) {
             );
             return;
         }
-        throw new IllegalStateException(String.format("User %s does not exist", uid));
+        throw new IllegalStateException(String.format("User %s does not exist", uname));
+    }
+
+    @Override
+    public void alterPassword(final String uname, final JsonObject info) {
+        if (this.users().map(yaml -> yaml.yamlMapping(uname) != null).orElse(false)) {
+            YamlMappingBuilder all = Yaml.createYamlMappingBuilder();
+            final YamlMapping users = this.users().get();
+            for (final YamlNode node : users.keys()) {
```

Comment:
Lines 127 and 129 invoke the same method: this.users()
I would to store result and reuse it.